components/css/global/_top.scss

```diff
@@ -23,22 +23,56 @@
 			a{
 				margin:0 auto;
 				width:50%;
+			} 
+		}
+		.svgLinkContainer {
+			  display: block;
+			  position: relative;
+			  height: 1px;
+			  width: 100%;
+			  padding-bottom: 29.95839%;
+			  z-index: 2;
+			  &::after{
```

Comment:
What is this `after` doing?

Reply:
That's for the gray line between the logo and tagline. Instead of having a border left for the tagline, it's now on the logo so the line will be longer.

Reply:
I thought that was down on line 137 - a `before` pseudo.

Remove the `after` styles here doesn't seem to have any effect on the design.

Reply:
Ok, I'll remove it than.

Reply:
This makes the whole logo clickable, if I remove it, you can only click on the logo below it, instead of the whole logo.